src/api/Feed.js

```diff
@@ -622,29 +617,54 @@ class Feed extends Base {
             }
         };
 
+        const v2Promises = [
+            versionsPromise,
+            currentVersionPromise,
+            commentsPromise(),
+            tasksPromise,
+            appActivityPromise,
+            annotationsPromise,
+        ];
+
+        const fetchV2FeedItems = async promises => {
+            return Promise.all(promises).then(
+                ([versions: ?FileVersions, currentVersion: ?BoxItemVersion, ...feedItems]) => {
+                    const versionsWithCurrent = currentVersion
+                        ? this.versionsAPI.addCurrentVersion(currentVersion, versions, this.file)
+                        : undefined;
+                    return sortFeedItems(versionsWithCurrent, ...feedItems);
+                },
+            );
+        };
+
+        const compareV2AndUaaFeedItems = async (uaaFeedItems, uaaResponse) => {
+            fetchV2FeedItems(v2Promises).then(v2FeedItems => {
+                const transformedV2FeedItems = collapseFeedState(v2FeedItems);
+                const transformedUAAFeedItems = collapseFeedState(uaaFeedItems);
```

Comment:
nit: inconsistency of `uaa` between this variable `transformedUAAFeedItems` and the function name `compareV2AndUaaFeedItems`